sdk/web-pubsub/ci.yml

```diff
@@ -1,5 +1,3 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
That's removed by our automation tool. @colawwj Could you add it back? I will change the tool accordingly so that it won't remove this comment in the future.

Reply:
Can you point me to the code that is writing this? I'd like to understand what automation we have that mutate these.